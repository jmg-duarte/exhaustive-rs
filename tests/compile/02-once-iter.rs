use exhaustive::c;



fn main() {
    c![n * n];
}