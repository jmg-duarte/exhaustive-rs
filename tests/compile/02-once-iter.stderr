error: unexpected end of input, expecting `for`
 --> $DIR/02-once-iter.rs:6:5
  |
6 |     c![n * n];
  |     ^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
